src/test/java/org/arquillian/tests/pagetests/BlogPageTest.java

```diff
@@ -92,4 +92,17 @@ public void should_redirect_to_new_announcement_if_banner_is_present() throws Ex
             .verify()
             .hasAnnouncementBanner(false);
     }
+
+    @Test
+    public void should_ignore_release_notes_for_non_release_blog_posts() throws Exception {
+        mainPage.menu()
+                .navigate().to("Blog");
+
+        blogPage.cloudTag()
+                .navigate().to("nonrelease");
+
+        blogPage.blogContent()
+                .verify()
+                .hasReleaseNotes();
```

Comment:
I don't understand the flow of this test

- we navigate to Blog
- we navigate to non-release (I guess we click on the tag)
   - this leads us to a page with several blogs
- we check if the content has release notes then (content of what?)

Reply:
I was hoping that you can propose some improvement in this code :)